src/iocore/cache/CacheHosting.cc

```diff
@@ -78,12 +78,12 @@ CacheHostMatcher::PrintFunc(void *opaque_data)
   d->Print();
 }
 
-// void CacheHostMatcher::AllocateSpace(int num_entries)
+// void CacheHostMatcher::AllocateSpace(uint32_t num_entries)
 //
 //  Allocates the HostLeaf and Data arrays
 //
 void
-CacheHostMatcher::AllocateSpace(int num_entries)
+CacheHostMatcher::AllocateSpace(uint32_t num_entries)
 {
   // Should not have been allocated before
   ink_assert(array_len == -1);
```

Comment:
There are some suspicious asserts throughout this file similar to the ones in ControlMatcher.cc.